packages/core/content-releases/admin/src/pages/ReleaseDetailsPage.tsx

```diff
@@ -314,13 +320,28 @@ export const ReleaseDetailsLayout = ({
   );
 };
 
+const GROUP_BY_OPTIONS = [
+  {
+    label: 'Content Type',
+    value: 'contentType',
+  },
+  {
+    label: 'Locale',
+    value: 'locale',
+  },
+  {
+    label: 'Action',
+    value: 'action',
+  },
```

Comment:
Action isn't in the figma design (even though it's supported in the API)

Reply:
You're right 😅, I think we talked about this in some meeting. In any case I discussed it with Yannis and we can keep it because it's already implemented it and add value to the group by filter